modules/internal/ChapelRange.chpl

```diff
@@ -806,11 +806,11 @@ module ChapelRange {
     return arg2 == arg1(arg2);
   }
 
-  proc ==(r1: range(?), r2: range(?)) param
+  operator ==(r1: range(?), r2: range(?)) param
```

Comment:
I suspect these `range` ones should be operator methods too but that should probably be future work.